src/gd_kotlin.cpp

```diff
@@ -232,18 +166,19 @@ void GDKotlin::init() {
 
     BridgesManager::get_instance().initialize_bridges(env, class_loader);
 
-    if (is_gc_activated) {
+    if (jvm_config.is_gc_activated) {
+        bool is_gc_force_mode{jvm_config.is_gc_force_mode};
```

Comment:
Couldn't this be inlined below?

Reply:
We use it twice, here and 5 lines below.